src/components/PatientProfilePage/PatientProfilePage.tsx

```diff
@@ -264,6 +381,42 @@ const PatientProfilePage = ({
             value={formFields.notes}
             isValidated={false}
           />
+          {transportConfirmed === true && (
+            <>
+              <Typography
+                variant="h5"
+                style={{ marginBottom: 24, marginTop: 30 }}
```

Comment:
I've a noticed several occurrences of non px values, do we want to be consistent?